package.json

```diff
@@ -39,8 +39,9 @@
 		"gettext-parser": "^4.0.3",
 		"husky": "^3.0.2",
 		"identity-obj-proxy": "^3.0.0",
-		"jest": "^27.4.5",
+		"jest": "29.5.0",
 		"jest-image-snapshot": "^6.1.0",
+		"jest-jasmine2": "29.5.0",
```

Comment:
`jest-jasmine2` is no longer included with Jest and it has [to be installed separately](https://jestjs.io/docs/28.x/upgrading-to-jest28#test-runner).